pr_agent/servers/github_app.py

```diff
@@ -359,6 +359,50 @@ def should_process_pr_logic(body) -> bool:
     return True
 
 
+def normalize_file_path(file_path: str) -> str:
+    """
+    Normalize a file path to be relative to the repository root.
+    - If the path starts with '/', remove it
+    - If the path is empty, return empty
+    """
+    if not file_path:
+        return ""
+    # Remove leading slash if present
+    if file_path.startswith('/'):
+        file_path = file_path[1:]
+    return file_path
+
+
+def extract_rag_edit_args(issue_body: str) -> Tuple[str, str]:
+    """
+    Extract file path and RAG prompt from the issue body.
+    
+    Format expected:
+    /rag_edit <file_path> <RAG prompt>
```

Comment:
We should have our own internal prompt for the specific case we want to handle, for these commands and issue handling. Similar to how we have ones for reviewing code, writing a PR description, we should have one for editing a file, tyring to solve an issue, etc.  We can have user input on the commands that we pass into these prompts but we need a base prompt for these scenarios.